seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/scheduler/JobScheduler.java

```diff
@@ -17,6 +17,10 @@
 
 package org.apache.seatunnel.engine.server.scheduler;
 
+import com.hazelcast.cluster.Address;
+
 public interface JobScheduler {
     void startScheduling();
+
+    Address queryTaskGroupAddress(long taskGroupId);
```

Comment:
Please don't save TaskGroup address from scheduler, because scheduler will never use when all of the tasks deploy complete. If Master Node done and another Master Node active, the running job will not be scheduled again.

Reply:
Done